v3/src/v2/codap-v2-data-set-types.ts

```diff
@@ -14,8 +14,7 @@ export interface ICodapV2Attribute {
   // TODO_V2_IMPORT_EXPORT
   categoryMap?: any
   _categoryMap?: any
-  // TODO_V2_IMPORT_EXPORT
-  colormap?: any
+  colormap?: Record<string, string>
```

Comment:
```suggestion
  colormap?: DICategoryColorMap
```
And you will need to move the definition of `DICategoryColorMap` into this file to avoid an import cycle.